File: v1_python_version/test_case_set.py
from device import Device
from link_station import LinkStation

class TestCaseSet:
	def __init__(self):
		self.link_stations = [
			LinkStation(0, 0, 10),
			LinkStation(20, 20, 5),
			LinkStation(10, 0, 12)
		]

		self.devices = [
			Device(0, 0),
			Device(100, 100),
			Device(15, 10),
			Device(18, 18)
		]

File: v2_typescript_version/testCaseSet.ts
import { LinkStation } from "./linkStations";
import { Device } from "./device";

export class TestCaseSet {
	linkStations: LinkStation[];
	devices: Device[];

	constructor() {
		this.linkStations = [
			new LinkStation(0, 0, 10),
			new LinkStation(20, 20, 5),
			new LinkStation(10, 0, 12)
		]

		this.devices = [
			new Device(0, 0),
			new Device(100, 100),
			new Device(15, 10),
			new Device(18, 18)
		]
	}
}